simconnect-sdk/src/domain/client_event.rs

```diff
@@ -63,6 +85,24 @@ impl ClientEventRequest {
             Self::AxisLeftBrakeSet => "AXIS_LEFT_BRAKE_SET\0".as_ptr() as *const c_char,
             Self::AxisRightBrakeSet => "AXIS_RIGHT_BRAKE_SET\0".as_ptr() as *const c_char,
             Self::ParkingBrakes => "PARKING_BRAKES\0".as_ptr() as *const c_char,
+            // Comm Radios
+            Self::Com1RadioStbySet => "COM_STBY_RADIO_SET_HZ\0".as_ptr() as *const c_char,
+            Self::Com2RadioStbySet => "COM2_STBY_RADIO_SET_HZ\0".as_ptr() as *const c_char,
+            Self::Com3RadioStbySet => "COM3_STBY_RADIO_SET_HZ\0".as_ptr() as *const c_char,
```

Comment:
I think if we're going to be opinionated about it, I don't think we should leave open the option for the BCD encoded event personally, even though we have the capability. Hz should be pretty obvious to those trying to set a COM or NAV radio in my opinion. Actually now that I think about it, with our new BCD convertion they'd supply Hz either way.

If you want it to fully spelly "Standby" that's fine, I can make that change.

Reply:
It would be good to leave it open for someone to implement it in the future if they want to. I can't see any use cases, but the users of this library always manage to surprise me.
Consistency is more important, however. All the other options in this enum are 1:1 mappings to sim connect and we direct people to read SC's documentation.